Resources/Private/Partials/ControlBar.html

```diff
@@ -42,16 +42,31 @@ <h1><f:link.page pageUid="{rootPageId}" title="DFG-Viewer" class="dfgviewer-logo
         </nav>
       </div>
 
-      <div class="metadata-wrapper">
-        <div class="tx-dlf-metadata">
-            <f:cObject typoscriptObjectPath="plugin.tx_dlf_metadata" />
-        </div>
-        <div class="offcanvas-toggle" title="<f:translate key='metadata' extensionName='dfgviewer' />"></div>
-      </div>
+      <div class="control-bar-content">
+          <div class="metadata-wrapper">
+              <div class="tx-dlf-metadata">
+                  <div class="control-bar-container metadata-basic">
+                      <h3>
+                            <span
+                                class="sidebar-heading-label">{f:translate(key:'metadata', extensionName:'dfgviewer')}</span>
```

Comment:
Is there some reason why it is split in 2 lines?

Reply:
No there is no reason. I changed this to one line. Thx :)